llama-index-core/llama_index/core/schema.py

```diff
@@ -253,10 +283,24 @@ class BaseNode(BaseComponent):
         default_factory=list,
         description="Metadata keys that are excluded from text for the LLM.",
     )
-    relationships: Dict[NodeRelationship, RelatedNodeType] = Field(
+    relationships: Dict[
+        Annotated[NodeRelationship, EnumNameSerializer],
+        RelatedNodeType,
+    ] = Field(
         default_factory=dict,
         description="A mapping of relationships to other node information.",
     )
+    metadata_template: str = Field(
+        default=DEFAULT_METADATA_TMPL,
+        description=(
+            "Template for how metadata is formatted, with {key} and "
+            "{value} placeholders."
+        ),
+    )
+    metadata_separator: str = Field(
```

Comment:
you miiiight need an alias to make this fully backwards compatible (there was a typo in the old name, whoops)

Reply:
I didn't remove the old one, see https://github.com/run-llama/llama_index/pull/16962/files/6d05cb2d87c28fba59abb8e5692839af3a95faf5#diff-36053ee3f29c57f74fbd3d7bf920633fadb93f8eff85251d95dd707bb4d929b9R597